### ISSUE TITLE
Unifying Studio Text Editors with Latest TinyMCE Version

### ISSUE THREAD

**Issue by santiagosuarezedunext** [openedx/platform-roadmap#358]
### Abstract

This initiative aims to standardize the text editor experience within Open edX Studio by converting the "Course Updates" page to MFE technology and integrating the latest version of TinyMCE. This change will ensure consistency in the text editing experience and enhance content creation for course creators and instructors.

### Context & Background

Currently, Open edX Studio employs a different text editor in the "Course Updates" section, leading to inconsistencies in functionality and user experience. Converting this page to MFE technology and integrating the latest version of TinyMCE will streamline content creation and enhance user satisfaction:

- Current Course Updates Text Editor: 
![Screenshot 2024-05-22 at 7 05 18 PM](https://github.com/openedx/platform-roadmap/assets/88672889/f65146e2-e9a8-4d2f-8337-43bbae294c52)

### Scope & Approach

The scope of this initiative involves converting the "Course Updates" page to MFE technology and installing or updating TinyMCE to ensure uniformity within Open edX Studio.

### Value & Impact

This initiative will enable users to leverage the latest tools available in TinyMCE, including custom features such as the ability to embed iframes seamlessly through a graphical interface. By integrating these advanced functionalities in the "Course Updates" section, users will have access to a comprehensive suite of editing tools, empowering them to create rich course content effortlessly.

### Milestones and/or Epics

Milestone 1: Conversion of Course Updates to MFE Technology
Milestone 2: Integration of Latest TinyMCE Version in Studio -> Content -> Course Updates

### Named Release

Sumac

### Timeline

This initiative is expected to be completed an released in Sumac version

### Proposed By

edunext

**Comment by jmakowski1123** [comment]
Brad Brown - PM at 2U

**Comment by sarina** [comment]
In Redwood we are debuting the Studio MFE, and new features should be developed on master. Have you checked out the Redwood test instance,  because I thought the Studio MFE used the TinyMCE editor in those places

**Comment by marcotuts** [comment]
Relatedly - Once this page is migrated to the MFE I assume some updates to the page styling / etc would be warranted since it has to be converted anyways. Updating to the latest TinyMCE makes sense but other (likely minor) updates to the underlying page would be helpful as well

**Comment by santiagosuarezedunext** [comment]
@marcotuts You're right. Is there any plan or person responsible for updating that page to MFEs?

**Comment by sarina** [comment]
I think the implementation of this proposal would be to convert this page to the MFE.

**Comment by santiagosuarezedunext** [comment]
@sarina Yes you are right I am going to change the description.
